controller/cache/compliance.go

```diff
@@ -246,6 +252,35 @@ func (m CacheMethod) GetRiskScoreMetrics(acc, accCaller *access.AccessControl) *
 		s.Groups.Groups++
 	}
 
+	// Calculate pod Vuls
+	podVulsMap := make(map[string]*podVuls) // podName => podVuls
+	for _, cache := range wlCacheMap {
+		wl := cache.workload
+		if !acc.Authorize(wl, nil) {
+			continue
+		}
+		if common.OEMIgnoreWorkload(wl) {
+			continue
+		}
+		if cache.platformRole != "" && disableSystem {
+			// skip system containers
+			continue
+		}
+		if !wl.Running || cache.scanBrief == nil {
+			continue
+		}
+
+		// do not re-sum existing pod vuls
+		if wl.ShareNetNS != "" {
```

Comment:
here it's
```
if wl.ShareNetNS != "" {
   // put podVulsMap
}
```
but in line 311 it's 
```
if wl.ShareNetNS == "" {
   // get podVulsMap
}
```
it looks like the calculated podVulsMap will never be used, or it will never be calculated, for a workload?

Reply:
After scan is done, the critical, high, med Vuls of each containers will be added into workload cache scan brief. However, this GetRiskScoreMetrics API will collect pod Vuls instead of each containers.
For the calculation part, 
`if wl.ShareNetNS != "" {
   // put podVulsMap
}`
I'll sum all containers Vuls into the map but not for the pod(avoid double calculate because it will be saved into cache later).